vendor/k8s.io/kube-openapi/pkg/util/proto/validation/types.go

```diff
@@ -210,7 +210,7 @@ func (item *primitiveItem) VisitPrimitive(schema *proto.Primitive) {
 		}
 	case proto.Number:
 		switch item.Kind {
-		case proto.Number:
+		case proto.Integer, proto.Number:
```

Comment:
this commit should have the glide.yaml change too

Reply:
I think it does 

```
{
"ImportPath": "k8s.io/kube-openapi",
"Rev": "61e04a5be9a6"
},
```

Link: https://github.com/openshift/origin/blob/487752ab83a9c39deb6b3b6292aceb94f2e89f0c/vendor/k8s.io/kubernetes/Godeps/Godeps.json#L1606

Reply:
Does github hide it here? I only see the types.go change here in this commit.

Reply:
or do we completely rely on glide reading Godeps? Then it's fine.